client/src/pages/positions/MyCounterparts.js

```diff
@@ -0,0 +1,26 @@
+import AppContext from "components/AppContext"
+import Fieldset from "components/Fieldset"
+import {
+  mapPageDispatchersToProps,
+  PageDispatchersPropType
+} from "components/Page"
+import PositionTable from "components/PositionTable"
+import React, { useContext } from "react"
+import { connect } from "react-redux"
+
+const MyCounterparts = ({ pageDispatchers }) => {
```

Comment:
> If we don't use pageDispatchers in this component, why do we declare it?

Probably (but @maradragan or @VassilIordanov should know for sure) because the initial idea was to do pagination here as well, but the verdict was that it wasn't needed (people don't have more than a small handful of counterparts anyway).
Good catch, I'll remove that bit (seeing as @maradragan is on holiday).